src/flask_container_scaffold/util.py

```diff
@@ -15,3 +19,28 @@ def load_yaml(filename='config.yml', logger=None):
     with open(filename, 'r') as file_handle:
         config = parse(file_handle, logger=logger)
     return config
+
+def load_cfg(conf_file):
+    """
+    Load a cfg file
+
+    :param filename str: A cfg/ini file to be parsed
+    :return: A dictionary fomred out of the cfg file data
+    :raises: configparser.MissingSectionHeaderError
+    """
+    config = configparser.ConfigParser()
+    # The following setting tells the parser not to alter the keys it
+    # has read in.  The default behavior converts these to lowercase:
+    # https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.optionxform
+    config.optionxform = lambda option: option
+    config.read(conf_file)
+    return _parse_cfg(config)
+
+def _parse_cfg(config):
+    config_dict = {}
+    settings_dict = {}
+    for section in config.sections():
+        for key in config[section]:
+            settings_dict[key] = config[section][key]
+        config_dict.update({section: settings_dict})
```

Comment:
This doesn't look right to me. settings_dict doesn't get reinitialised in the loop so if there are multiple sections, the latter section will also end up with all the keys from the first sections, I think.

Reply:
I think you are right.  I'll move initialization of settings_dict inside the for loop so it is new each time we get to a new section.